flink-core/src/main/java/org/apache/flink/configuration/Configuration.java

```diff
@@ -757,7 +757,7 @@ else if (configOption.hasFallbackKeys()) {
 	}
 
 	@Override
-	public <T> WritableConfig set(ConfigOption<T> option, T value) {
+	public <T> Configuration set(ConfigOption<T> option, T value) {
```

Comment:
This is necessary as a workaround to make the builder pattern with `addConfiguration()` work, right?

Reply:
I wouldn't call it a workaround, but yes :)

It is so that we can just inline the instantiation and setting like:

```
public void addConfiguration(Configuration config);

 addConfiguration(
   new Configuration()
     .set(...)
     .set(...)
     .set(...)
 )
```

Without this change it would have to be:

```
Configuration config = new Configuration();
config
     .set(...)
     .set(...)
     .set(...)
 addConfiguration(config)
```